src/App.js

```diff
@@ -1,25 +1,91 @@
-import logo from './logo.svg';
+import React, { Component } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Grid from './Components/Grid/Grid.js';
+import CharacterPage from './Components/CharacterPage/CharacterPage';
+import Header from './Components/Header/Header';
+import Fellowship from './Components/Fellowship/Fellowship';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component  {
+  constructor() {
+    super();
+    this.state = {
+      characters: [],
+      team: [],
+      selectedPlayer: ''
+    }
+  }
+
+
+  fetchData = async () => {
+    try {
+      const response = await fetch('https://the-one-api.dev/v2/character', {
```

Comment:
I like how you used async await here. I still haven't really tried it, so I'm nervous.